pkg/system/phase3_connecting.go

```diff
@@ -141,15 +140,8 @@ func (r *Reconciler) InitNBClient() error {
 			return fmt.Errorf("mgmt port not ready yet")
```

Comment:
I guess this check is not really making much sense now...

Reply:
correct

Reply:
correct